libexec/rtld-elf/libmap.c

```diff
@@ -103,7 +103,7 @@ lmc_parse_file(const char *path)
 	char *lm_map;
 	struct stat st;
 	ssize_t retval;
-	int fd, saved_errno;
+	int fd, saved_errno __unused;
```

Comment:
I'm guessing upstream always builds with the dbg() messages enabled or some such which is why we don't notice this one upstream?

Reply:
Yes. We split out separate rtld-elf*-debug that have messages available, whilst rtld-elf* have them compiled out. Somewhat dubious a downstream diff these days...